### ISSUE TITLE
Here's a new site I'd like you to consider blocking

### ISSUE THREAD

**Issue by ehuntercommer** [grantwinney/hide-comments-everywhere#84]
Here's a new site I'd like you to consider blocking:

https://www.cbc.ca

(please include any other relevant details)

**Comment by ehuntercommer** [comment]
Actually... it looks like the extension isn't currently working on any site. Youtube comments are back as well. I'll close this, and investigate if it's something on my end.

**Comment by grantwinney** [comment]
It's not something on your end. I broke the addon with a recent update. The temp fix is to go to the options page, click Save on the "blacklist" textbox (you don't even need to type anything in it), and try again.

The _real_ fix (basically, just doing an extra null check in case there's no blacklist yet) is waiting to go out, but the store team always takes a few days to get releases out. Ugh. 😧

I tried posting updates everywhere I could think of that people might see (including a new issue on here), but yeah.. not really a good way to tell people, oops I broke everything. 🙄

I'll re-open this for now, just in the hopes someone else might see it if they come here.
